src/core/sys/posix/sys/socket.d

```diff
@@ -594,14 +594,24 @@ version( CRuntime_Glibc )
 
     enum : uint
     {
-        MSG_CTRUNC      = 0x08,
-        MSG_DONTROUTE   = 0x04,
-        MSG_EOR         = 0x80,
         MSG_OOB         = 0x01,
         MSG_PEEK        = 0x02,
+        MSG_DONTROUTE   = 0x04,
+        MSG_CTRUNC      = 0x08,
+        MSG_PROXY       = 0x10,
         MSG_TRUNC       = 0x20,
+        MSG_DONTWAIT    = 0x40,
+        MSG_EOR         = 0x80,
         MSG_WAITALL     = 0x100,
-        MSG_NOSIGNAL    = 0x4000
+        MSG_FIN         = 0x200,
+        MSG_SYN         = 0x400,
+        MSG_CONFIRM     = 0x800,
+        MSG_RST         = 0x1000,
+        MSG_ERRQUEUE    = 0x2000,
+        MSG_NOSIGNAL    = 0x4000,
+        MSG_MORE        = 0x8000,
+        MSG_WAITFORONE  = 0x10000,
+        MSG_FASTOPEN    = 0x20000000
```

Comment:
According to https://sourceware.org/git/?p=glibc.git;a=blob;f=sysdeps/unix/sysv/linux/bits/socket.h;h=c3fbb2110296273c1275f9a0c37ae753428dff7c;hb=HEAD, the complete definition would be:
```D
enum : uint
{
    MSG_OOB          = 0x01,
    MSG_PEEK         = 0x02,
    MSG_DONTROUTE    = 0x04,
    MSG_TRYHARD      = MSG_DONTROUTE,
    MSG_CTRUNC       = 0x08,
    MSG_PROXY        = 0x10,
    MSG_TRUNC        = 0x20,
    MSG_DONTWAIT     = 0x40,
    MSG_EOR          = 0x80
    MSG_WAITALL      = 0x100
    MSG_FIN          = 0x200,
    MSG_SYN          = 0x400,
    MSG_CONFIRM      = 0x800,
    MSG_RST          = 0x1000,
    MSG_ERRQUEUE     = 0x2000,
    MSG_NOSIGNAL     = 0x4000,
    MSG_MORE         = 0x8000,
    MSG_WAITFORONE   = 0x10000,
    MSG_BATCH        = 0x40000,
    MSG_ZEROCOPY     = 0x4000000,
    MSG_FASTOPEN     = 0x20000000,
    MSG_CMSG_CLOEXEC = 0x40000000
}
```